### ISSUE TITLE
Feature request: Movable panel/windows

### ISSUE THREAD

**Issue by potholiday** [yhat/rodeo#491]
Currently rodeo panel/windows are fixed and we cant move it or resize it like in  RStudio

> Editor	                        Enviroment/History
> Console	                    File explorer/Plot

I would like to move my Panels around  and if possible resize a selected panel to full screen

> Console                      Enviroment/History
> Editor                          File explorer/Plot

**Comment by danielgwilson** [comment]
I second this—my R studio setup is so much nicer than the default. Also, console theme should match global theme.

<img width="1792" alt="screen shot 2016-10-14 at 01 28 53" src="https://cloud.githubusercontent.com/assets/11265967/19376666/e0c29fae-91ad-11e6-8629-e3d6e8b867d5.png">

**Comment by TakenPilot** [comment]
You should be able to drag the non-editor tabs around in https://github.com/yhat/rodeo/releases/tag/v2.4.8

Could you try it out and tell me what you think?

**Comment by danielgwilson** [comment]
@TakenPilot 
Yeah that's solid, I like that better than R actually, where you have to set it in preferences. You should still be able to move the editor, though.

Also, it does not save preferences for where tabs were or how each tab is sized. This is not expected behavior; tabs should remember positions and sizes on close.

**Comment by TakenPilot** [comment]
Yeah, it should.  I'm using a thing called Redux to save the application state, but I haven't got to the part where I'm saving the application state between Rodeo sessions completely yet.  

It's on the roadmap that Colin and I are talking about, because it would save me so much time in testing the application...

**Comment by skykistler** [comment]
Has this been fixed? This really killed rodeo for me.. beautiful app but the layout just does not make much sense to me..
